### ISSUE TITLE
Filter API Search with a language parameter

### ISSUE THREAD

**Issue by luddaniel** [ontoportal/ontoportal-project#56]
**Describe the bug**
While using API `/search` query parameter `lang` or `language` is not taken in account.

**To Reproduce**
Example : https://data.agroportal.lirmm.fr/search?pagesize=1&q=phylloxera&language=en
Result:
```json
"prefLabel": [
	"Phylloxera",
	"Phylloxera",
	"حشرة الأوراق الجافة",
	"Phylloxera",
	"Phylloxera",
	"Phylloxera",
	"Phylloxera",
	"Phylloxera",
	"Phylloxera",
	"Phylloxera",
	"Phylloxera",
	"포도뿌리혹벌레속",
	"Phylloxera",
	"فیلوکسرا",
	"Phylloxera",
	"Phylloxera",
	"Phylloxera",
	"Phylloxera",
	"Phylloxera",
	"フィロキセラ属",
	"倭蚜属",
	"Phylloxera",
	"Phylloxera",
	"फाइलोक्सेरा",
	"Phylloxera"
],
```

**Expected behavior**
`prefLabel` and `synonym` should only contain the specified language.
Also, it should be possible to specify many languages like `language=en,fr`

**Comment by luddaniel** [comment]
Seems fixed by https://github.com/ontoportal-lirmm/ontologies_api/pull/82
But I still experience weird stuff : 

Without parameter `language`
![Sans langue](https://github.com/user-attachments/assets/ad0140e2-d1d6-492e-8e82-780bb614e4a5)
___
With `language=fr`
![Avec langue fr](https://github.com/user-attachments/assets/2fa68e7a-162d-4eae-a611-1a7ba0953327)

What do you think about it @syphax-bouazzouni ?

**Comment by syphax-bouazzouni** [comment]
> Seems fixed by [ontoportal-lirmm/ontologies_api#82](https://github.com/ontoportal-lirmm/ontologies_api/pull/82) But I still experience weird stuff :
> 
> Without parameter `language` ![Sans langue](https://private-user-images.githubusercontent.com/83018819/359416887-ad0140e2-d1d6-492e-8e82-780bb614e4a5.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQxNDc1OTUsIm5iZiI6MTcyNDE0NzI5NSwicGF0aCI6Ii84MzAxODgxOS8zNTk0MTY4ODctYWQwMTQwZTItZDFkNi00OTJlLThlODItNzgwYmI2MTRlNGE1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODIwVDA5NDgxNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTU2ZTE3NTFhYTg4OGI2OGU3MTI2ZmFkMzQ1ZjA1OTM2NGZjZTZhMDZjYzJhNDI4ZWYwNmJlZjEwZGM3MTUwMDImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.wEhgdlYy5NlCph6kKIR6Y2MnMsBs30uHdW6DRbdoCFI)
> 
> With `language=fr` ![Avec langue fr](https://private-user-images.githubusercontent.com/83018819/359417375-2fa68e7a-162d-4eae-a611-1a7ba0953327.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQxNDc1OTUsIm5iZiI6MTcyNDE0NzI5NSwicGF0aCI6Ii84MzAxODgxOS8zNTk0MTczNzUtMmZhNjhlN2EtMTYyZC00ZWFlLWE2MTEtMWE3YmEwOTUzMzI3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODIwVDA5NDgxNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJjODZmNDJlMGFlYzAxYmViNmY4ODEyMDgwY2Y2OTBjODZkZWMxZDhiMDJjZDI1OGZkYmQ0NWJiYWJiM2I5ZDEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.f464T3tJrSKG87VIcusShF1XsLiYwF9i3Jph5zP7M3E)
> 
> What do you think about it @syphax-bouazzouni ?

@luddaniel, when you do `lang=fr` it means you filter/search only in values labeled with French tag.  It does not mean you display the results in french.

**Comment by luddaniel** [comment]
Trouble was with `language` that does not behave like `lang`.
I have a really good behavious with https://data.agroportal.lirmm.fr/search?q=poulard&lang=en,fr
Having both search and result in french and english.
Good job, this closes the issue.
